components/EntryForm.js

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components";
+import DatePicker from "react-datepicker";
+import { useState } from "react";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default function EntryForm() {
+  const [startDate, setStartDate] = useState(new Date());
+  const [selectedOption, setSelectedOption] = useState("Car");
+
+  const handleDropdownChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  function handleFormSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    event.target.reset();
+    console.log(data);
+  }
+
+  return (
+    <>
+      <h1 className="calculator-title">Calculator</h1>
+      <section>
+        <h2 className="section-title">Measure your impact</h2>
+        <Form onSubmit={handleFormSubmit}>
+          <h3 className="form-title">Enter your journey: </h3>
+          <label htmlFor="date">Date: </label>
+          <DatePicker
+            showIcon
+            selected={startDate}
+            onChange={(date) => setStartDate(date)}
+            dateFormat="dd/MM/yyyy"
+          />
+          <label htmlFor="start">From: </label>
+          <input id="start" name="start" />
+          <label htmlFor="destination">To: </label>
+          <input id="destination" name="destination" />
+          <label htmlFor="km">Km: </label>
+          <input id="km" name="km" />
+          <label htmlFor="transport">
+            Transport:
+            <select value={selectedOption} onChange={handleDropdownChange}>
+              <option value="option1">Car</option>
+              <option value="option2">Plane</option>
+              <option value="option3">Train</option>
+              <option value="option4">Bicycle</option>
```

Comment:
You should set the individual values to something more semantic like "car" instead of "option1".

Reply:
Thanks for your feedback! I will fix it.